victorLi/assignments/treesDFS/lc113/lc113.js

```diff
@@ -12,6 +12,21 @@
 //  / |   | \
 // 4  5   2  7
 
+/*traverse using DFS
+  if path = target then push stack into allPaths array
```

Comment:
When do you push stack into allPaths? More detail on this approach would help make it clearer how you code is attempting to satisfy the prompt.